### DISCUSSION TITLE
Get current value of variable with hyprctl?

### DISCUSSION THREAD

**Discussion by mylinuxforwork** [hyprwm/Hyprland#4714] [General]
Hi all. 

I‘m currently working on a script that uses hyprctl to modify some settings of Hyprland. To initiate the script, I would like to use current values of the keywords e.g, get value of border_size and use it in a script.

Is there a hyprctl command available for getting current values?

Thanks,

Stephan

**Comment by fufexan** [comment]
`hyprctl getoption` as described in [the wiki](https://wiki.hyprland.org/Configuring/Using-hyprctl/#info).

**Reply by mylinuxforwork** [reply]
I missed that. Thank you so much.
